src/client.ts

```diff
@@ -439,6 +440,8 @@ in an environmental variable named FAUNA_SECRET or pass it to the Client\
     ) {
       headerObject["x-last-txn-ts"] = this.#lastTxnTs;
     }
+
+    headerObject["X-Driver-Env"] = getDriverEnv();
```

Comment:
let's cache this once at construction time so we don't have to farm this on every request.